app/gui2/src/stores/suggestionDatabase/index.ts

```diff
@@ -9,35 +9,21 @@ import { defineStore } from 'pinia'
 import { LanguageServer } from 'shared/languageServer'
 import { markRaw, ref, type Ref } from 'vue'
 
-export class SuggestionDb {
-  _internal = new ReactiveDb<SuggestionId, SuggestionEntry>()
-  nameToId = new ReactiveIndex(this._internal, (id, entry) => [[entryQn(entry), id]])
-  parent = new ReactiveIndex(this._internal, (id, entry) => {
+export class SuggestionDb extends ReactiveDb<SuggestionId, SuggestionEntry> {
+  nameToSuggestionId = new ReactiveIndex(this, (id, entry) => [[entryQn(entry), id]])
```

Comment:
Do we need `Suggestion` part of the name?

Reply:
not really... i was kinda assuming it'd make things clearer, but after thinking about it, i guess it's already clear enough. (plus, adding `Suggestion` makes it *over twice* as long)